subsys/fb/cfb.c

```diff
@@ -330,6 +330,7 @@ int cfb_framebuffer_init(const struct device *dev)
 	struct display_capabilities cfg;
 
 	api->get_capabilities(dev, &cfg);
+	api->blanking_off(dev);
```

Comment:
This is not necessary, it is up to the app to disable it, see https://github.com/zephyrproject-rtos/zephyr/blob/1c8f1cd590e10dcc349ebd8649c313293e29209c/samples/subsys/display/cfb/src/main.c#L40

Reply:
Thak you for reviewing.

I misunderstood some design policies.
I rewritten this patch to modify only the sample.